reset_security_server.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Script should be run as sudo
+# Usage reset_security_server.sh -n=server(s) -h=path/to/hosts.txt -a=path/to/ansible
+# Usage example: reset_security_server.sh -n=ss3,ss4 -h=../../X-Road/ansible/hosts/lxd_hosts.txt -a=../../X-Road/ansible
+
+if [ "$#" -ne 3 ]; then
+    echo "Usage: reset_security_server.sh -n=name(s) -h=hosts.txt -a=ansible_folder"
+    exit 1
+fi
+
+for i in "$@"
+do
+case $i in
+    -n=*|--name=*)
+    NAME="${i#*=}"
+    shift
+    ;;
+    -h=*|--hosts=*)
+    HOSTS="${i#*=}"
+    shift
+    ;;
+    -a=*|--ansible=*)
+    ANSIBLE="${i#*=}"
+    shift
+    ;;
+    *)
+        echo "Usage: reset_security_server.sh -n=name(s) -h=hosts.txt -a=ansible_folder"
+        exit 1
+    ;;
+esac
+done
+
+names=$(echo "$NAME" | tr "," "\n")
```

Comment:
``bash`` is used as interpreter for reset_security_server.sh -- it also allows to define functions, so that I would recommend using them to have the main script in logical blocks, separating e.g. command line argument parsing, actual reset functionality, maybe something else. That way there is more overview of what is going on and the scripts are easier to change (plus some functions might be able to be reused, if we end up needing to have more scripts of our own).

Reply:
Functions are used now